release/models/aft/.spec.yml

```diff
@@ -2,7 +2,9 @@
   docs:
     - yang/aft/openconfig-aft.yang
     - yang/aft/openconfig-aft-types.yang
+    - yang/aft/openconfig-aft-summary.yang
   build:
     - yang/network-instance/openconfig-network-instance.yang
     - yang/aft/openconfig-aft-network-instance.yang
+    - yang/aft/openconfig-aft-summary.yang
```

Comment:
Also add in network-instance's .spec.yml as otherwise it won't show up in the online documentation due to openconfig-aft only having augmented nodes.

Reply:
Done